demos/reference-check/pages/api/drivers.ts

```diff
@@ -0,0 +1,19 @@
+// Next.js API route support: https://nextjs.org/docs/api-routes/introduction
+import type { NextApiRequest, NextApiResponse } from 'next';
+import { findDrivers, insertDriver } from '@/server/db';
+
+const handler = (req: NextApiRequest, res: NextApiResponse) => {
+  if (req.method === 'POST') {
+    const user = insertDriver({
+      ...req.body,
+    });
+
+    res.status(201).json(user);
+
+    return;
+  }
+
+  res.status(200).json(findDrivers());
```

Comment:
do we want to wrap this with a `GET` method?

Reply:
Yes.